docs/mp/integrations/jedis.adoc

```diff
@@ -157,3 +176,8 @@ Injection without a `@Named` annotation is also possible:
 ----
 In this case, the properties for JedisPoolConfig and JedisPool that can be set will start wih `redis.clients.jedis.JedisPoolConfig.default` and `redis.clients.jedis.JedisPool.default` respectively.
 
+== References
+
+* link:{helidon-github-tree-url}/examples/integrations/cdi/jedis[Helidon Jedis Usage Examples]
+* link:{jedis-base-url}/jedis/JedisPool.html[JedisPool] javadoc
+* link:https://github.com/redis/redis[Redis]
```

Comment:
This doesn't match the Jedis Github link at the top of this document.

Reply:
Fixed